_includes/releases/babelfish-2.1.2.md

```diff
@@ -0,0 +1,26 @@
+- Babelfish Version: 2.1.2
+- PostgreSQL Server Version: 14.3
+- Download source distributions:
+  - [BABEL_2_1_2__PG_14_3.zip](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases/download/BABEL_2_1_2__PG_14_3/BABEL_2_1_2__PG_14_3.zip)
+  - [BABEL_2_1_2__PG_14_3.tar.gz](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases/download/BABEL_2_1_2__PG_14_3/BABEL_2_1_2__PG_14_3.tar.gz)
+- Babelfish Compass
+  - [Download](https://github.com/babelfish-for-postgresql/babelfish_compass/releases)
+- Date: November 8, 2022
+
+## Overview
+
+Babelfish 2.1.2 contains the functionality and features of versions 2.1.0 and 2.1.1, as well as:
+
+- a fix for critical issues in Babelfish due to incorrect handling of user input for some application features.
+- a fix for error handling in sp_prepare calls which can cause a server crash when a large number of parameters are sent by the application. Babelfish currently supports a maximum of 100 parameters for a procedure or function.
+- a fix for error handling in SSL/TLS handshake for some client drivers.
+
+
+Babelfish does not support an upgrade path to version 2.1.2 from Babelfish 1.x.x.
```

Comment:
We should additionally provide the output of below command for checking if Babelfish upgrade was successful:
```bash
babelfish_db[124386]# SELECT sys.serverProperty('BabelfishVersion') AS BabelfishVersion;
 babelfishversion 
------------------
 2.1.2
(1 row)
```

Reply:
Updated...